proposals/lambda-method-group-defaults.md

```diff
@@ -145,31 +190,44 @@ only been supported since C# 10, so only code which has been written since then
 ### Grammar and Parser Changes
 This enhancement requires the following changes to the grammar for lambda expressions.
 ```diff
-explicit_anonymous_function_parameter
+ explicit_anonymous_function_parameter_list
++    : explicit_anonymous_function_parameters
++    | explicit_anonymous_function_parameters ',' parameter_array
++    | parameter_array
++    ;
++
++explicit_anonymous_function_parameters
+     : explicit_anonymous_function_parameter
+       (',' explicit_anonymous_function_parameter)*
+     ;
+
+ explicit_anonymous_function_parameter
 -    : anonymous_function_parameter_modifier? type identifier
 +    : anonymous_function_parameter_modifier? type identifier default_argument?
-    ;
+     ;
 ```
 No changes to the grammar are necessary for method groups since this proposal would only change their semantics.
 
 ### Binder Changes
 
 #### Synthesizing New Delegate Types
-As with the behavior for delegates with `ref` or `out` parameters, a new natural type is generated for each lambda or method group defined with any default parameter values.
+As with the behavior for delegates with `ref` or `out` parameters, a new natural type is generated for each lambda or method group defined with any default or `params` parameters.
 Note that in the below examples, the notation `a'`, `b'`, etc. is used to represent these anonymous delegate types.
 
 ```csharp
 var addWithDefault = (int addTo = 2) => addTo + 1;
-// internal delegate int b'(int arg0 = 2);
+// internal delegate int a'(int arg0 = 2);
```

Comment:
>int arg0 [](http://example.com/codeflow?start=28&length=8)

`int arg`

Reply:
Right, thanks. Fixed also in other places: bc44898d4a61c1efe13e00c486dc3bb03323c040